src/shared/components/Button/styles.ts

```diff
@@ -0,0 +1,82 @@
+import styled from 'styled-components';
+import Colors from '../../colors';
```

Comment:
You could use absolute paths: 
`import Colors from 'shared/colors';`

Reply:
done

Reply:
done